content/katas/fibonacci sequence/_index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Fibonacci sequence
+pre: "<b>1. </b>"
+weight: 5
+---
+
+
+ **What is fibonacci sequence?**
+
+> Fibonacci Sequence: is the series of numbers where the next number is found by adding up the n numbers before it.
+> e.g. in this case n = 2:
+> sequence: 0, 1, 1, 2, 3, 5, 8, 13, 21, 34, ...
+
+**1.** 
+write a function that takes in a number argument [n] and returns fibonacci sequence of the given number [n], "series of values" 
```

Comment:
rephrase:
Write a function, `fibonacci(n)` that takes in a number/int argument (n) and returns the fibonacci sequence from the 1st term of the sequence till the nth term of the sequence